src/app/components/MappedMessage.js

```diff
@@ -17,7 +16,7 @@ function mapGlobalMessage(intl, msgKey, values) {
 
 function FormattedGlobalMessage({ messageKey, ...rest }) {
   const message = globalStrings[messageKey];
-  return <FormattedMessage {...message} {...rest} />;
+  return <FormattedMessage {...message} {...rest} />; // eslint-disable-line @calm/react-intl/missing-attribute
```

Comment:
Note to reviewers: I am not sure how to resolve this, or if its necessary to. Any guidance would be appreciated